pyvcloud/vcd/system.py

```diff
@@ -51,3 +51,30 @@ def create_org(self, org_name, full_org_name):
             RelationType.ADD,
             EntityType.ADMIN_ORG.value,
             org_params)
+
+    def get_org(self, org_name):
+        """
+        Retrieve an organization.
+        :param org_name: name of the org to be retrieved.
+        :return: Org record.
+         """  # NOQA
+        orgs = self.client.get_org_list()
+        for org in [o for o in orgs.Org if hasattr(orgs, 'Org')]:
+            if org.get('name') == org_name:
+                return org
+        raise Exception('org \'%s\' not found' % org_name)
+
+    def delete_org(self, org_name, force=None, recursive=None):
+        """
+        Delete an organization.
+        :param org_name: name of the org to be deleted.
+        :param force: pass force=True  along with recursive=True to remove an
+        organization and any objects it contains, regardless of their state.
+        :param recursive: pass recursive=True  to remove an organization
+        and any objects it contains that are in a state that normally allows.
+        removal.
+         """  # NOQA
+        org = self.get_org(org_name)
+        org_href = org.get('href').replace('/api/org/',
```

Comment:
I noticed that System object has admin_href variable. Please check if we can use it instead of replace call.